app/Http/Controllers/TeamMembersController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Alexa\Models\Team;
+use App\Alexa\Models\User;
+use Illuminate\Http\Request;
+use App\Exceptions\TeamException;
+use Illuminate\Support\Facades\DB;
+use App\Rules\User\EligibleForNewTeam;
+
+class TeamMembersController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @param  int  $teamId
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index($teamId)
+    {
+        $team = DB::transaction(function () use ($teamId) {
+            return auth()->user()->ownerTeam()
+                ->with('users')
+                ->findOrFail($teamId);
+        });
+
+        return view('team_members.index', compact('team'));
+    }
+
+    /**
+     * Store the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $teamId
+     *
+     * @return array
+     */
+    public function store(Request $request, $teamId)
+    {
+        try {
+            DB::transaction(function () use ($request, $teamId) {
+                $team = auth()->user()->ownerTeam()->findOrFail($teamId);
+
+                $this->validate($request, [
```

Comment:
It can't be moved (as a while) because it calls the database. 
Do you think that `required` and `string` should be moved?

Reply:
Yes, because maybe later we may need a `CreateRequest` for some new validations and forget that this is being done here and add a double `required` and `string` validation, don't you agree?

Reply:
Agree 👍